Makefile

```diff
@@ -553,3 +553,16 @@ tilt: generate manifests tilt-prepare
 
 tilt-clean:
 	rm -rf .tiltbuild
+
+.PHONY: local-setup
```

Comment:
since this is outside of the scope of this pr, can we have a separate pr for this please?